### ISSUE TITLE
[New Feat Propos] missing-super: Add new check to validate a return after a super

### ISSUE THREAD

**Issue by moylop260** [PyCQA/pylint#1157]
We missing many times a return after a overwrite method.

```python
class A:
  def method_x(self):
     ...
     return something

class B(A):
  def method_x(self):
     res = super(self, B).method_x()
     ...  # custom changes
     # missing `return res`
```

There is exceptions like as: `__init__, setUp, tearDown` using a configuration variable to add them.

The question is: Is a good idea for this project add this check?

**Comment by PCManticore** [comment]
I don't understand what you mean. Do you refer to res not being returned from method_x?

**Comment by moylop260** [comment]
Yes I do.

The original method `return something` but the second one is not.
IMHO a inherited method should return the original or modify value from inherited method.

What do you think?

**Comment by rogalski** [comment]
IMHO it's hard to imagine a checker for this that'll be simultaneously helpful and false-positive-free.

I think you underestimate how hard is to come up with formal rules of _modify (return) value from inherited method_.

Maybe I want to inherit a class to use exception raising instead of returned value as error indicator.

```
def method(self):
    return_value = super().method()
    if return_value in KNOWN_ERROR_CODES:
        raise LibraryException(return_value)

```

What if returned value from super call is `(value, error_code)`? Is return value from super used? Not used? Partially used?

**Comment by moylop260** [comment]
Yeah, we have a particular case that should be implement from custom plugin.
Thanks for reply
